### ISSUE TITLE
Gradle build under Gradle 8 has inconsistent configuration

### ISSUE THREAD

**Issue by cyrille-artho** [javapathfinder/jpf-core#422]
When using
`./gradlew build -info`, we get a number of warnings like this one:
```
The configuration classpath is both consumable and declarable. This combination is incorrect, only one of these flags should be set.
```
This should be fixed so future versions of Gradle will not cause undetermined behavior.

**Comment by gaurangkudale** [comment]
It should display `consumable` because it's used to specify the dependencies that the project needs to build and run correctly.

**Comment by cyrille-artho** [comment]
Do you know how to declare a task as `consumable`? The only information I could find was to create a separate local build with a local maven archive, which seems to be rather complicated to achieve something that worked out of the box in older versions of Gradle.

**Comment by gaurangkudale** [comment]
I'll check for other options in official documentation of Gradle and update it.

**Comment by gaurangkudale** [comment]
This issue is fixed in `gradle version 8.4`
Reference Link : https://github.com/gradle/gradle/issues/24197
